database/mongoDb.js

```diff
@@ -0,0 +1,214 @@
+const mongoose = require('mongoose');
+const dbName = 'zillow';
+const config = require('../config');
+
+
+var featuresTable;
+
+const mockFeatures = require('./mockFeatures');
+const mockInterior = require('./mockInterior');
+const features = require('./createMockData');
+const fs = require('fs');
+var documentCount = 0;
+
+
+  var createConnection = function (cb) {
+
+    db = mongoose.connect(`mongodb://${config.host}:27017/${dbName}`, {useNewUrlParser: true});
+    let schema = new mongoose.Schema({
+      house_id: {
+        type: Number,
+        unique: true
+      },
+      type: String,
+      year_built: Number,
+      heating: String,
+      cooling: String,
+      parking: String,
+      lot: Number,
+      days_on_zillow: Number,
+      bedrooms: Number,
+      bathrooms: Number,
+      interiorheating: String,
+      interiorcooling: String,
+      appliances: String,
+      kitchen: String,
+      flooring: String,
+      sqft: Number
+    }, { _id : false });
+    featuresTable = mongoose.model('features', schema);
+    console.log('Connected');
+    featuresTable.countDocuments({}, function(err, count) {
+      if (err) { return handleError(err) } //handle possible errors
+

```

Comment:
handleError is not in this review, so not sure what it handles. This is a great use case for having a top level catch and throw in function

mdn.io/catchthrow